partials/new_device_login.html

```diff
@@ -1,4 +1,8 @@
 <% extends "partials/base/base_text_only.html" %>
 
 <% block heading %>{{t "New sign-in to Firefox"}}<% endblock %>
-<% block text_primary %>{{{ device }}}<br/>{{ timestamp }}<% endblock %>
+<% block text_primary %>{{{ device }}}<br/>
+<span style="display:block;">IP address: {{ ip }}</span>
```

Comment:
Yees, I would format this like:

```
{{t "IP address: %(ip)s" }}
```

Again, we have to ensure the `t` helper function can do interpolation.